docs/_template.html

```diff
@@ -33,7 +33,7 @@
       <div id="sidebar-logo" class="d-flex justify-content-center p-5">
       <a href="./index.html" ><img id="fsdocs-logo" src="{{root}}/{{fsdocs-logo-src}}" /></a>
 </div>
-      <div class="list-unstyled ps-0 text-white fs-6">
+      <div id=sidebar-generation" class="list-unstyled ps-0 text-white fs-6">
```

Comment:
Nitpick, but as nothing is no longer generated you should consider to change the name of this identifier.